specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/CommonDefinitions.json

```diff
@@ -1176,6 +1180,20 @@
         }
       }
     },
+    "KeyVaultProperties": {
+      "description": "Key vault reference using managed identity.",
+      "type": "object",
+      "properties": {
+        "identity": {
+          "description": "Resource ID of a managed identity to authenticate with Azure Key Vault, or System to use a system-assigned identity.",
+          "type": "string"
+        },
+        "keyVaultUrl": {
```

Comment:
>keyVaultUrl [](http://example.com/codeflow?start=9&length=11)

certificateURLInKeyVault?

Reply:
Again I would favor consistency throughout the Container App API here. We use `keyVaultUrl` in other places to refer to a KV secret that holds a certificate, so once users have seen it in one place it should be familiar in others.

@njuCZ let's update the description to be a little clearer that this should be a URL to the secret in Key Vault: "URL pointing to the Azure Key Vault secret that holds the certificate."